frontend/src/metadata/cells/SliceCell.svelte

```diff
@@ -37,6 +37,11 @@
 		: false;
 
 	$: selected = $selections.slices.includes(slice.sliceName);
+	$: transferData =
```

Comment:
What if the slice name has a comma in it?

Reply:
will it be better if changing the comma to a random generated hash string as the delimiter?